builders/tooling/oracles/acurast.md

```diff
@@ -7,20 +7,22 @@ description: Learn how to use Acurast's decentralized serverless cloud to get re
 
 ## Introduction {: #introduction }
 
-[Acurast](https://acurast.com/){target=\_blank} is a trustless orchestration layer for decentralized computing capabilities. Among other features, Acurast enables you to interact with off-chain price feeds that can be accessed via API. Pricing data is processed through the Acurast network and accessible from EVM-compatible chains like Tanssi EVM Appchains via a [standard Chainlink Aggregator Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank}. 
+[Acurast](https://acurast.com/){target=\_blank} gives developers complete permisionless access to compute that is trustless, affordable and confidential for deploying their applications.
+
+One of the Acurast's use cases is to enable developers to deploy their own push/pull oracles, interacting with off-chain APIs to bring e.g., price feeds on-chain. Pricing data is confidentially processed through Acurast Processors, pushing data to smart contracts of EVM-compatible chains like Tanssi EVM Appchains to e.g., a [standard Chainlink Aggregator Interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank} smart contract.
 
 This tutorial will walk through a demo of [interacting with price feeds](#fetch-price-data) enabled by Acurast on the [demo Tanssi EVM-compatible Appchain](/builders/tanssi-network/networks/dancebox/demo-evm-containerchain/){target=\_blank}. You can also deploy your own price feeds to your Tanssi EVM-compatible Appchain. Please be advised that the steps shown in this tutorial are for demonstration purposes only - it's highly recommended that you contact the [Acurast](https://acurast.com/){target=\_blank} team directly as they can assist you with launching price feeds on your Appchain to ensure the integrity of the deployment process.
 
 ## What is Acurast? {: #what-is-acurast }
 
-The Acurast network has three main stakeholders: consumers, processors, and the orchestrator. 
+Acurast, the decentralized serverless cloud.
+
+The cloud is dominated by three main players, this centralization comes at a price: high cost of compute, exclusion of less developed parts of the world, and the hidden cost, loss of data ownership. Acurast is solving all three problems by bringing the convenience of a cloud - whilst giving users full confidentiality over their data - at a fraction of the cost.
```

Comment:
Please revise this to make it more informational and less like a pitch. We try to maintain a neutral tone in our docs rather than taking a side.